engine/client/fragstats.c

```diff
@@ -424,30 +475,21 @@ void Stats_Evaluate(fragfilemsgtypes_t mt, int wid, int p1, int p2)
 		fragstats.clienttotals[p1].grabs++;
 		fragstats.totaltouches++;
 
-		if (u1 && p1 >= 0)
-		{
-			Stats_Message("You grabbed the flag\nflag grabs: %i (%i)\n", fragstats.clienttotals[p1].grabs, fragstats.totaltouches);
-		}
+		Stats_FlagMessage(ff_flagtouch, u1, p1, fragstats.clienttotals[p1].grabs, fragstats.totaltouches);
```

Comment:
I realize now that this is a behavioral change. The u1 case ought to be left intact, and the u1 part of Stats_FlagMessage ought to be removed, and this would be `else if (p1 >= 0)` instead.